### PR TITLE
Translation of the website to arabic

### PR DESCRIPTION
**Pull request by mftatfcyclone**
I translated all words on the website from English to Arabic. Furthermore, I would like to contribute more to translation or programming in order to publish the website with Arabic language.

### CODE REVIEW

File: src/i18n/ar.json [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    "peru": "بيرو",
+    "philippines": "الفلبين",
+    "poland": "بولندا",
+    "portugal": "البرتغال",
+    "qatar": "قطر",
+    "romania": "رومانيا",
+    "russia": "روسيا",
+    "saintkittsandnevis": "سانت كيتس ونيفيس",
+    "sanmarino": "سان مارينو",
+    "saudiarabia": "Saudi Arabia",
```
Comment (liquidx) [inline src/i18n/ar.json:176]: Is there a reason why this one is not translated?

**Comment by liquidx** [comment]
Would it be possible to add this change into change #338 so that we can test the whole change in one PR?

File: src/i18n/ar.json (hunk 1 above)
Comment (mftatfcyclone) [reply src/i18n/ar.json]: I am sorry for that, I have corrected it

**Comment by mftatfcyclone** [comment]
Is there any additional task required for this language? I think all files are ready.

**Comment by mftatfcyclone** [comment]
Is there any additional task required for this language? I think all files are ready.

**Comment by vlumi** [comment]
> Is there any additional task required for this language? I think all files are ready.

The file `i18n/index.js` would need to be updated to refer to the added language, too, matching how the other languages are configured. Also, country translations apart from "westerdam" should be removed, as they come now from another module, with Arabic support.

While with this everything is translated, the flow of everything on the page is still left-to-right -- is that not a problem?

**Pull request closed without merging**